zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -4180,37 +4157,45 @@ components:
       type: object
       properties:
         sort:
+          description: Sort field criteria.
           type: array
           items:
             allOf:
               - $ref: "#/components/schemas/SearchQuerySortRequest"
         page:
+          description: Pagination criteria.
           allOf:
             - $ref: "#/components/schemas/SearchQueryPageRequest"
           type: object
     SearchQueryPageRequest:
       type: object
       properties:
         from:
+          description: The index to start searching from.
```

```suggestion
          description: The index of items to start searching from.
```

A little more clarity, inspired by https://camunda.slack.com/archives/C06UKS51QV9/p1731939210511069.